docs/source/model-serving.rst

```diff
@@ -11,19 +11,32 @@ Model Serving Basics
 Background
 ++++++++++
 
-XXX
+Most of the attention paid to machine learning has been devoted to algorithm development.
+However, models are not created for the sake of their creation, they are created
+to be put into production. Usually when people talk about taking a model “to
+production,” they mean performing inference. A complete inference solution seeks to provide serving, 
+monitoring, and updating functionality. This chapter will focus on the model serving, there are advantages 
+about model serving as following,
+
+* Helping to standardize model serving across orgs with unified data plane and pre-built model servers
+* A single way to deploy, monitor inference services/server, and scale inference workload
+* Dramatically shortening time for data scientists to deploy models to production
 
 Overview
 ++++++++
 
-.. figure:: ./figs/model-serving-architecture.png
```

Comment:
Remove this file from git, since it is not used anymore?

Reply:
remove msa figure